app/build.gradle.kts

```
@@ -41,6 +41,8 @@ android {
 
 dependencies {
 
+    implementation("androidx.compose.material:material-icons-extended:1.7.0")
```

toml 파일 이용해서 import 해봅시다.
```suggestion
    implementation(libs.compose.material.icons.extended)
```